gapic/src/main/com/google/gapid/perfetto/models/GpuInfo.java

```diff
@@ -93,20 +94,22 @@ public int bufferCount() {
   public static class Queue {
     public final int id;
     public final long trackId;
+    public final String name;
     public final int maxDepth;
 
-    public Queue(int id, long trackId, int maxDepth) {
+    public Queue(int id, long trackId, String name, int maxDepth) {
```

Comment:
This changes more than what your description says. This changes the name of all the queue tracks.

Reply:
Updated description.